src/main/java/nl/ordina/jobcrawler/scrapers/JobBirdScraper.java

```diff
@@ -0,0 +1,231 @@
+package nl.ordina.jobcrawler.scrapers;
+
+import lombok.extern.slf4j.Slf4j;
+import nl.ordina.jobcrawler.model.Vacancy;
+import nl.ordina.jobcrawler.model.VacancyURLs;
+import nl.ordina.jobcrawler.service.ConnectionDocumentService;
+import org.jsoup.nodes.Document;
+import org.jsoup.nodes.Element;
+import org.jsoup.select.Elements;
+import org.springframework.stereotype.Component;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+
+/*
+MylerVacancyScraper.java takes care of scraping vacancies from myler.nl
+At this moment getVacancies() returns a list with Vacancies that only exist of a vacancyURL and the broker. No details available (yet).
+ */
+
+@Slf4j
+@Component
+public class JobBirdScraper extends VacancyScraper {
+
+    //private static final String SEARCH_URL = "https://www.jobbird.com/nl/vacature?s=java&rad=30&ot=date";
```

Comment:
Commented code should be deleted. Or it should be made clear why it's still there.

Reply:
adapted, explanation about the way the url is created